sampledb/models/migrations/action_type_add_template.py

```diff
@@ -0,0 +1,81 @@
+# coding: utf-8
+"""
+Create default action types.
+"""
+
+import os
+from ..actions import ActionType
+from ..languages import Language
+
+MIGRATION_INDEX = 69
+MIGRATION_NAME, _ = os.path.splitext(os.path.basename(__file__))
+
+
+def run(db):
+    # Add TEMPLATE to default action types
+    existing_action_type_ids = [
+        action_type[0]
+        for action_type in db.session.execute("""
+                    SELECT id
+                    FROM action_types;
+                """).fetchall()
+    ]
+    if ActionType.TEMPLATE in existing_action_type_ids:
+        return False
+
+    action_type_template = {
+        'id': ActionType.TEMPLATE,
+        'admin_only': False,
+        'show_on_frontpage': True,
```

Comment:
```suggestion
        'show_on_frontpage': False,
```
I would suggest not showing template actions on the front page by default.